packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -1377,14 +1379,24 @@ export default ({
   }
 
   const fetchLimits = function * ({
-    currency
-  }: ReturnType<typeof A.fetchSBFiatEligible>) {
+    cryptoCurrency,
+    currency,
+    side
+  }: ReturnType<typeof A.fetchLimits>) {
     try {
       yield put(A.fetchLimitsLoading())
-      const limits: ReturnType<typeof api.getSwapLimits> = yield call(
-        api.getSwapLimits,
-        currency
-      )
+      let limits
+      if (cryptoCurrency && side) {
+        limits = yield call(
+          api.getSBLimits,
+          currency,
+          ProductTypes.SIMPLEBUY,
+          cryptoCurrency,
+          side
+        )
+      } else {
+        limits = yield call(api.getSwapLimits, currency)
+      }
```

Comment:
we now have calling of same endpoint but with passing along diff parameters, once when we move SM calls from `packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/index.tsx` then we can clean this up